src/stdlib_stats_distribution_PRNG.fypp

```diff
@@ -1,16 +1,15 @@
 #:include "common.fypp"
 module stdlib_stats_distribution_PRNG
     use stdlib_kinds, only: int8, int16, int32, int64
+	use stdlib_error
     implicit none
     private
     integer, parameter :: MAX_INT_BIT_SIZE = bit_size(1_int64)
-    integer(int64), save :: st(4), si = 614872703977525537_int64
+    integer(int64), save :: st(4), si = 614872703977525537_int64 ! default seed
     logical, save :: seed_initialized = .false.
```

Comment:
Just a style nitpick: I would prefer to split this into two lines and add a comment that `st(4)` is the internal state of the PRNG.